docs/GUIDE.en.md

```diff
@@ -243,7 +243,17 @@ near account \
     network-config testnet
 ```
 
-You will be redirected to the browser for authorization. After successful authorization in _[NEAR Wallet](https://wallet.near.org/)_, you need to return to the terminal and enter your login.
+You will be redirected to the browser for authorization.  
+Default wallet url is https://app.mynearwallet.com/ (for testnet - https://testnet.mynearwallet.com/). But if you want to change to a different wallet url, you can use `--walletUrl` option:
+```txt
+near account \
+    import-account \
+    using-web-wallet \
+    --wallet-url 'https://wallet.testnet.near.org/'\
+    network-config testnet
```

Comment:
Looking at this command I realized that it would be great to have ad-hoc ways to alter `network-config`, so adding optional parameters to `NetworkConfig` makes total sense, and then the command would be:

```txt
near account \
    import-account \
    using-web-wallet \
    network-config testnet \
    --wallet-url 'https://wallet.testnet.near.org/'
```

Alternatively, we may consider implementing "Wallet Selector" when instead of direct URL it will ask if one wants to use "NEAR Wallet" or "MyNearWalllet" or "Meteor Wallet" or ... and then the command would be:

```txt
near account \
    import-account \
    using-web-wallet \
    "MyNearWallet" \
    network-config testnet
```

This path with Wallet selectors is harder to implement and maintain, so I would at least start with the first option (extending `network-config` with optional parameters)